### PR TITLE
Zh hans.lproj

### PR DESCRIPTION
**Pull request by gujiaming2022**
Simplified Chinese

### CODE REVIEW

**Review by L1ghtmann — CHANGES_REQUESTED** [review]
Thank you so much for doing this! Would you mind retaining the file format from the base `en.lproj` (i.e., the spacing, section comments, header, etc). Once that is done, I'll merge. Thanks!

**Comment by gujiaming2022** [comment]
Is the format correct?

**Comment by L1ghtmann** [comment]
Yup, looks great! Did you want to credit yourself at the top? If not, that's fine too.

**Review by L1ghtmann — APPROVED** [review]

File: zh-Hans.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,89 @@
+//
+//	Translation by Lightmann
```
Comment (L1ghtmann) [inline zh-Hans.lproj/Localizable.strings:2]:
```suggestion
//	Translation by gujiaming2022
```

**Pull request merged**